exec/java-exec/src/main/java/org/apache/drill/exec/planner/sql/SchemaUtilites.java

```diff
@@ -77,6 +77,22 @@ public static SchemaPlus findSchema(final SchemaPlus defaultSchema, final String
     return findSchema(defaultSchema, schemaPathAsList);
   }
 
+  /**
+   * Utility function to get the commonPrefix schema between two supplied schemas.
```

Comment:
Could you please add example in Java doc?

Reply:
done.